plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/jdbc/TestIcebergJdbcCatalogConnectorSmokeTest.java

```diff
@@ -198,6 +198,13 @@ public void testRenameSchema()
                 .hasMessageContaining("renameNamespace is not supported for Iceberg JDBC catalogs");
     }
 
+    @Test
+    @Override
+    public void testDeleteRowsConcurrently()
+    {
+        // TODO https://github.com/trinodb/trino/issues/21862 Fix flaky test
```

Comment:
Do we know why this is failing?

Reply:
Nope. I guess the engine was busy because of concurrent tests.